airflow/providers/common/sql/operators/sql.py

```diff
@@ -203,6 +203,7 @@ class SQLExecuteQueryOperator(BaseSQLOperator):
     :param split_statements: (optional) if split single SQL string into statements. By default, defers
         to the default value in the ``run`` method of the configured hook.
     :param return_last: (optional) return the result of only last statement (default: True).
+    :param print_to_log: (optional) if true operator output will be print to log (default: False).
```

Comment:
WDYT about naming this param "show_output_in_logs" instead? Same vein for naming as "show_return_value_in_logs" as the `@task` decorator. And, I guess it's not _technically_ printing.